### ISSUE TITLE
Dipole correction still needed?

### ISSUE THREAD

**Issue by sjtuzhanglei** [shankar1729/jdftx#347]
Dear Shankar,

For implicit solvation model, do we still need to do a dipole correction for asymmetric slab?

I assume the dipole is compensated by the solvation (dielectric+ionic) response, am I right?

Best,
Lei

**Comment by shankar1729** [comment]
Yes, that's right: no need for dipole correction because the solvent+ions
screening exponentially attenuates the spurious dipole interactions between
periodic images.
